build.proj

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="dir.props" />
+  <Import Project="$(ToolsDir)\$(BuildToolsPackageName).$(BuildToolsVersion)\lib\packages.targets" />
```

Comment:
Done -- makes more sense to have this in a variable and to have the variable in a centralized place.